maintainers/release-process.md

```diff
@@ -159,6 +148,30 @@ release:
 
 ## Creating a point release
 
+* Checkout.
+
+  ```console
+  $ git checkout XX.YY-maintenance
+  ```
+
+* Determine the next patch version.
+
+  ```console
+  $ export VERSION=XX.YY.ZZ
+  ```
+
+* Update release notes.
+
+  ```console
+  $ ./maintainers/release-notes
```

Comment:
This could be one line like above

```suggestion
  $ VERSION=X.YY.ZZ ./maintainers/release-notes
```

Reply:
I _think_ Eelco would have exported here in practice, while in the other place that's even less obvious.
I'm fine either way. Maybe @edolstra could decide?